### DISCUSSION TITLE
The controlled end does not have the highest authority

### DISCUSSION THREAD

**Discussion by SereinYun** [rustdesk/rustdesk#8763] [Q&A]
When I use version 1.2.6 for remote operation, the controlled end cannot move the cursor and has the highest authority.
But two days ago, during the operation, the controlled end could move the mouse freely while being controlled and end the remote connection at any time. How can this function be restored?
The controlled end cannot perform disconnection operations while being controlled, which poses a significant safety hazard!
Seeking help from experts

**Comment by rustdesk** [comment]
two days ago?

**Reply by SereinYun** [reply]
When I am remotely operating, the controlled end cannot move the mouse and cannot close the remote connection,Because I haven't used it for a long time, probably two days ago. During remote operation, the controlled end can still use the mouse. When the controlled end doesn't want to be controlled, it can directly disconnect. However, today during remote operation, the controlled end informed me that its mouse cannot be moved when controlled, nor can it close the connection. Only when I actively disconnect the remote control will the controlled end resume use.

**Reply by SereinYun** [reply]
If the controlled end does not have the highest authority during remote control, then as long as the control end connects to the controlled end, all permissions of the controlled end will be transferred and the connection cannot be disconnected,Seeking help from experts

**Comment by rustdesk** [comment]
Close to favor https://github.com/rustdesk/rustdesk/issues/40
